docs/content/guides/getting-started/introduction.md

```diff
@@ -8,62 +8,118 @@ react:
-  </Link>
-  <Link href="../javascript-data-grid/angular-simple-example" hide-latest-version class="row-item">
-    <Img class="integration-framework-logo" src="/img/pages/introduction/angular.svg" alt="Angular logo" />
-      <h3>Angular</h3>
-  </Link>
-  <Link href="../javascript-data-grid/vue-simple-example/" hide-latest-version class="row-item">
-    <Img class="integration-framework-logo" src="/img/pages/introduction/vue.svg" alt="Vue logo" />
-      <h3>Vue</h3>
-  </Link>
-  <Link href="../javascript-data-grid/vue3-simple-example/" hide-latest-version class="row-item">
-    <Img class="integration-framework-logo" src="/img/pages/introduction/vue.svg" alt="Vue 3 logo" />
-      <h3>Vue 3</h3>
-  </Link>
-</div>
+::: only-for react
+- [React demo](https://codesandbox.io/s/handsontable-react-data-grid-hello-world-app-forked-16c9gw)
 :::
 
-## What's new
+Then, move on to [connecting](@/guides/getting-started/binding-to-data.md) your data and [configuring](@/guides/getting-started/configuration-options.md) Handsontable's built-in features. For more complex implementations, use Handsontable's [API](@/api/introduction.md).
 
-- [Release notes](@/guides/upgrade-and-migration/release-notes.md)
-- [Blog](https://handsontable.com/blog)
-- [Twitter](https://twitter.com/handsontable)
+### Quickstart
+
+- [Install Handsontable](@/guides/getting-started/installation.md)
+- [Load data](@/guides/getting-started/binding-to-data.md)
+- [Save data](@/guides/getting-started/saving-data.md)
+- [Configure options](@/guides/getting-started/configuration-options.md)
+- [Define the size](@/guides/getting-started/grid-size.md)
+
+### Use popular features
+
+- [Cell types](@/guides/cell-types/cell-type.md)
+- [Formula calculations](@/guides/formulas/formula-calculation.md)
+- [Column filter](@/guides/columns/column-filter.md)
+- [Column groups](@/guides/columns/column-groups.md)
+- [Column summary](@/guides/columns/column-summary.md)
+- [Row parent-child](@/guides/rows/row-parent-child.md)
+- [Context menu](@/guides/accessories-and-menus/context-menu.md)
+
+### Customize the grid
+
+- [Create a custom renderer](@/guides/cell-functions/cell-renderer.md)
+- [Create a custom editor](@/guides/cell-functions/cell-editor.md)
+- [Create a custom validator](@/guides/cell-functions/cell-validator.md)
+- [Create a custom plugin](@/guides/tools-and-building/custom-plugins.md)
+- [Translate the UI](@/guides/internationalization/language.md)
+
+::: only-for javascript
+## Supported frameworks
+
+Handsontable supports popular JavaScript frameworks through wrappers. It also features a TypeScript declaration file that standardizes the API [methods](@/api/core.md), [hooks](@/api/hooks.md), and [options](@/api/options.md).
+
+Explore Handsontable's documentation for [React](../react-data-grid), [Angular](@/guides/integrate-with-angular/angular-installation.md), [Vue 2](@/guides/integrate-with-vue/vue-installation.md), and [Vue 3](@/guides/integrate-with-vue3/vue3-installation.md).
```

Comment:
Shouldn't the link to React be `../../react-data-grid`? The current one gives me 404

Reply:
@warpech – hm, on my side, it's the other way round: `../../react-data-grid` gives me 404, and `../react-data-grid` works: ![1](https://user-images.githubusercontent.com/29247866/188589469-9b372f78-decb-4c24-b885-c4f7925e62da.gif

Reply:
I think this is because you are using the dev server. In the production server, all links end with `/` and then `../../` is needed